### DISCUSSION TITLE
MC736429 and MC721851

### DISCUSSION THREAD

**Discussion by Brisso** [Micke-K/IntuneManagement#216] [Q&A] [answered]
Starting to get notifications from MS365 tenants regarding MC736429 and MC721851, and that PowerShell scripts may break. 

Do you know if this will cause issues for your app starting the 1st of May? 

Current Status: If you are using the Intune PowerShell application ID (d1ddf0e4-d672-4dae-b554-9d5bdfd93547), you will need to update your scripts before May 1 with a different Microsoft Entra ID registered application ID to prevent your PowerShell scripts from breaking.

**Comment by Micke-K** [comment, marked as answer]
Hello!

I just released a new version that will use Microsoft Graph app as default. 

This has been available for a long time in Settings. I probably should have changed the default app earlier though. 

Let me know how it goes if you test it 

Cheers

**Reply by Brisso** [reply]
Awesome work. We didn't realise that was in there as a setting; we tested today and the new version with the default is excellent.
